src/components/pages/Dashboard/DashboardContainer.js

```diff
@@ -15,7 +15,7 @@ function DashboardContainer() {
               // Currently for testing only
               [
                 {
-                  program: 'Shelter Support',
+                  program: 'Prevention',
```

Comment:
Any reason we decided to pick Shelter Support over Prevention in this? Were we trying to get a specific color for the dot and see if it worked?

Reply:
I changed it to get three different sample dummy data programs. to test.

Reply:
Sounds good! That makes sense.